.github/workflows/tests.yml

```diff
@@ -405,6 +405,40 @@ jobs:
           name: data-coverage
           path: ./coverage.xml
 
+  lightning-test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.11.0
+        with:
+          access_token: ${{ github.token }}
+
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.10'
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip && pip install wheel --upgrade
+          pip install -r requirements-ci.txt --upgrade
+          pip install -r requirements-dev.txt --upgrade
+          python -m pip install --index-url https://test.pypi.org/simple/ pennylane-lightning --pre --no-deps
+
+      - name: Install PennyLane
+        run: |
+          python setup.py bdist_wheel
+          pip install dist/PennyLane*.whl
+
+      - name: Run tests
+        run: |
+          python -m pytest pennylane/devices/tests --device=lightning.qubit

```

Comment:
Can we add something like `pl-device-test --device lightning.qubit --shots 10000 --skip-ops` or better here?